<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>customcreateoptions</class>
 <widget class="QDialog" name="customcreateoptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>125</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Volume Options</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Create Options Below</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbSet">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>90</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCancel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>90</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
